### PR TITLE
ENH: Large overhaul, including initial flowchart of the experiment

### PR DESCRIPTION
**Pull request by oesteban**
Several interesting features in addition to the flowchart.

- Preliminary session reorganized to improve readability
- Removed trailing spaces
- Added the `thanks` admonition
- Used the `quote` admonition for all verbal exchange scripts in acquisition.

### CODE REVIEW

**Comment by oesteban** [comment]
This is what the flowchart looks like:

sops_flowchart

It is done with *Mermaid*, which is an extension for markdown that allows generating charts from code.

You can play with it [here](https://mermaid.live/edit#pako:eNqFU01v2zAM_SuETh1QY81HL0FQIHGCLsCyBol3knNgbMYRakuCLG_z6v73ynbcJmm76WKD75HiI5-eWKRiYiO2T9Xv6IDGQjANZSjBnbzYJQb1AUK2iVBKIRNYK5WFrMXrEwtDkRVKNnldNDc93qSQ2Z5G-9x3wSLFs-iAr6dngSGf-_dnkVs-D44BkvEHDU4XD6uJ___OdkJpjHp8Nund3PjbS6Bf31wjsFwv3qEDPlkuHDp7hwz5JlidIKdNmh54nndXBUYkCZkK8h3flDLaqT_bjtJvKJAgv8ccJhLT8u_r7NyE2goh-0EJWvGL4Mr7ErLqYzlugG25zzW5ibYUsnxeEgQGo0cysKprtJQEu0v9Q73JFAZw5T-M3dDv-uOv9eekhcFnSUO4-kfOsZtdm_VzM61AOznOcFZkRSpAG8pJWmwW-R21VTpkRxGOCeMmcW4PZCTZyul5w95qZs7kGV8up4G3gYW0ZPYY0faC-m22XFTON7lOseQro0pnINVtoanREvu3nhYSNBpMU0o7PcNOT2tGuGhNu02EbHUoc6E8501l4vpFvYpi1ywjk6GI3Xt8qiuFzCVnFLKR-43RPNb-fnY8LKyqLcRG1hR0zQodo6WZQPccMjbaY5rT8wsGbSuu)

File: docs/data-collection/preliminary.md [hunk 1]
```diff
@@ -1,16 +1,128 @@
...
+    ga --->|"Channel 4 (O<sub>2</sub>)"| biopac3
+
+    sb --->|USB| pc1["Stimuli presentation Laptop ({{ secrets.hosts.psychopy | default("███") }})"]
+    pc1 <--->|Ethernet| et
+    pc1 --->|USB| modem[MMBT-S Interface]
+    pc1 --->|HDMI| display[Proyector]
+    modem --->|25-pin parallel| biopac4
+
+    BIOPAC --->|Ethernet| pc2["Physio-recording Laptop ({{ secrets.hosts.acqknowledge | default("███") }})"]
+```
```
Comment (oesteban) [inline docs/data-collection/preliminary.md]: This is the new flowchart.

File: docs/data-collection/scanning.md [hunk 2]
```diff
@@ -42,21 +42,22 @@
...
-    > 
-    > [Allow a few moments for the participant to breathe while you check the recordings]
-    >
-    > Okay, we seem to be able to track your respiration. Is the respiration belt too restraining?
-    > This is also a good moment to swallow, and to check your neck and head are in a comfortable position.
-    >
-    > For this first part, all you have to do is stay still; you can relax and close your eyes if it helps.
-    >
-    > Are you ready?
+    ???+ quote "Tell the participant that we are starting"
```
Comment (oesteban) [inline docs/data-collection/scanning.md:45]: Example of all script blocks changed to a `quote` admonition block (in this case it is foldable and it is first rendered expanded, as indicated by the `???+` prefix.

File: docs/stylesheets/extra.css [hunk 3]
```diff
@@ -54,4 +54,22 @@ span.flip {
...
+}
+.md-typeset .thanks > .admonition-title,
+.md-typeset .thanks > summary {
+  background-color: rgba(43, 155, 70, 0.1);
+}
+.md-typeset .thanks > .admonition-title::before,
+.md-typeset .thanks > summary::before {
+  background-color: rgb(43, 155, 70);
+  -webkit-mask-image: var(--md-admonition-icon--thanks);
+          mask-image: var(--md-admonition-icon--thanks);
```
Comment (oesteban) [inline docs/stylesheets/extra.css:74]: necessary for the new `thanks` admonition.

File: docs/data-collection/participant-prep.md [hunk 4]
```diff
@@ -116,13 +117,21 @@
...
+- [ ] Offer the participant a box to deposit everything they have in their pockets and all jewelry/hair accessories, and indicate the clothing to enter the scanning room:
 
-    > Before we continute, we need to make sure we do not introduce any dangerous object in the magnet room.
-    >
-    > [IF THEY NEED TO CHANGE INTO SCRUBS] Here you will find a changing room, I have prepared some scrubs for you. Please remove all your clothes and leave them in the changing room. Please keep your underwear on (if a woman, double check that **their undergarment DOES NOT contain any large metallic part** such as shaping guides, and request their removal if they do).
-    >
-    > [IF THEY CAN WEAR THEIR CLOTHES] Please deposit here all your belongings, your belt, your glasses, your jewelry and any accessories, piercings, etc. that you have on you.
+    ??? quote "*Dress code* inside the scanner **if they need to CHANGE INTO SCRUBS**"
+        Before we continute, we need to make sure we do not introduce any dangerous object in the magnet room.
```
Comment (esavary) [inline docs/data-collection/participant-prep.md]:
```suggestion
        Before we continue, we need to make sure we do not introduce any dangerous object in the magnet room.
```

File: docs/data-collection/preliminary.md [hunk 5]
```diff
@@ -1,16 +1,128 @@
...
+    The access panel also has a connector carrying the **trigger** signals generated by MR schemes, which indicate important events in the MRI acquisition (typically, one trigger pulse is generated for each new *repetition time* —TR—, e.g., with every fMRI volume).
+1. **Syncbox**.
+    A *NordicLabs Syncbox* receives TTL (transistor-transistor logic) triggers from the scanner.
+    This box can just forward the triggers converted into other formats and/or manipulate them (e.g., filter, generate, etc.).
+1. **Gas analyzer (GA)**.
+    The GA is a device that continuously measures the amount of two gases (CO<sub>2</sub> and O<sub>2</sub>) from a sample fed at the front of the device with a connected tube (this tube comes from inside of the Scanning Room and ends in the nasal cannula the participant is wearing, as shown in the graph).
+1. **BIOPAC**.
+    The BIOPAC is the main recording hub.
+    It directly receives analogical signals from the Scanning Room (for the case of the RB and the ECG).
+    Indirectly, it receives the analogical signal from the GA, and digital signals from the *Psychopy laptop* ({{ secrets.hosts.psychopy | default("███") }}).
```
Comment (esavary) [inline docs/data-collection/preliminary.md:59]:
```suggestion
    Indirectly, it receives the analogical signal from the GA, the triggers from the scanner and digital signals from the *Psychopy laptop* ({{ secrets.hosts.psychopy | default("███") }}).
```

File: docs/data-collection/scanning.md [hunk 6]
```diff
@@ -102,15 +103,15 @@
     ???+ quote "Participant of Cohort II"
 
         Hey [NAME], the next block is a bit long, around 30 minutes.
-        
+
         You can close your eyes and even sleep if you wish.
```
Comment (esavary) [inline docs/data-collection/scanning.md:107]: We may need to change this if we record ET data during DWI

File: docs/data-collection/preliminary.md [hunk 7]
```diff
@@ -1,16 +1,128 @@
...
+    sr4 ---> biopac2[ECG100C MRI]
+    sr5 ---> et[Eye Tracker PC]
+
+    ga --->|"Channel 3 (CO<sub>2</sub>)"| biopac3
+    ga --->|"Channel 4 (O<sub>2</sub>)"| biopac3
+
+    sb --->|USB| pc1["Stimuli presentation Laptop ({{ secrets.hosts.psychopy | default("███") }})"]
+    pc1 <--->|Ethernet| et
+    pc1 --->|USB| modem[MMBT-S Interface]
+    pc1 --->|HDMI| display[Proyector]
```
Comment (esavary) [inline docs/data-collection/preliminary.md]:
```suggestion
    pc1 --->|HDMI| display[Projector]
```

**Review by esavary — CHANGES_REQUESTED** [review]
LGTM, I just corrected some typos. I think the position of some images in pre-session. md is wrong, but I'll fix this in a next PR.

File: docs/data-collection/participant-prep.md (hunk 4 above)
Comment (oesteban) [reply docs/data-collection/participant-prep.md]: I wonder why codespell did not catch this?

File: docs/data-collection/preliminary.md (hunk 5 above)
Comment (oesteban) [reply docs/data-collection/preliminary.md:59]: It actually doesn't receive the trigger from the scanner, it is just one more signal.

File: docs/data-collection/scanning.md (hunk 6 above)
Comment (oesteban) [reply docs/data-collection/scanning.md:107]: Good catch!

Comment (oesteban) [reply docs/data-collection/scanning.md:107]: Can you file a new issue to remind us to change this?

Comment (esavary) [reply docs/data-collection/scanning.md:107]: Yes, no problem!

**Review by esavary — APPROVED** [review]

**Pull request merged**